text/0000-artifact-hub-integration.md

```diff
@@ -0,0 +1,166 @@
+registry service is a very important tool, but there is room for improvement.
+
+The mission of CNB is to enable **building**. Making artifacts discoverable is not necessarily part of the core mission,
+but it is a core supporting feature. Meanwhile, OCI artifact discoverability **is the core mission** of ArtifactHub.io.
+We should consider using this "off the shelf" product instead of enhancing our own registry. In this way, we can support
+the CNCF community while gaining a lot of features and dedicated user experience enhancements. We will benefit from
+contributors dedicated to artifact discoverability.
+
+Here is an [example of a helm chart](https://artifacthub.io/packages/helm/prometheus-community/prometheus) that is
+referenced on ArtifactHub.io. You can see here there are a lot of interesting features like security reports,
+vulnerabilities, usage stats, and all sorts of things.
+
+Now is a great time to interface with the ArtifactHub team. Buildpacks is very stable and ArtifactHub appears relatively
+stable as well.
+
+# What it is
+
+With (not insignificant) effort on our part, we could gain a lot of benefits from cooperating and integrating with
+ArtifactHub. For instance, ArtifactHub would provide us with a more extensive API, we can have data that signals to
+consumers which buildpacks are verified or officially supported by organizations, etc. We could have better navigation
+of Builders to Buildpacks and Run Images and even more metadata that would support buildpack adoption and signal CNB
+project maturity.
+
+# How it Works
+
+The ArtifactHub [roadmap](https://github.com/artifacthub/hub/blob/master/ROADMAP.md) outlines the direction of the
+project, which includes adding "support for more Cloud
+Native artifacts from CNCF projects." We should work with the ArtifactHub team to connect our registry and
+make the following Images/Artifacts first-class citizens in ArtifactHub:
+
+- CNB Component Buildpacks
+- CNB Builders
+- CNB Base Run Images
+
+# Migration
+
+[migration]: #migration
+
+There will not be a migration because we will not remove the existing registry.
+
+The official buildpacks registry is the [registry-index found at github](https://github.com/buildpacks/registry-index).
+This is the registry used by pack when it is identifying buildpacks referenced with the URN pattern of
+`urn:cnb:builder[:<id>[@<version>]]` or
+`urn:cnb:registry[:<id>[@<version>]]` ([ref](https://buildpacks.io/docs/for-app-developers/how-to/build-inputs/specify-buildpacks/)).
+This will remain in place.
+
+# Drawbacks
+
+[drawbacks]: #drawbacks
+
+We would need to consider changes to the ArtifactHub type schemas when modifying our spec. It is possible that the
+Artifact Hub artifact schemas and API could drift from the official RFC speec and implementation. We would need to
+consider backwards compatibility with the ArtifactHub implementation.
+
+# Alternatives
```

Comment:
Another alternative: we could also sunset our registry move to ArtifactHub entirely

Reply:
I will update

Reply:
After some more research, I have actually made this part of the main proposal.